### ISSUE TITLE
[Visual Requirement - Help - JavaScript]: Luminosity ratio is less than 4.5:1 for all links text on java script dialog. 

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2525]
**User Experience** 
If the luminosity ratio is less than 4.5:1 for all the text content then it will be difficult for the visually impaired user to see what is present on the screen.

**Test Environment** 
OS: RS2 Version 1703(OS Build 15063.483) 
Platform: Edge.
Tool: Color contrast Analyzer

**Repro Steps**
1-Navigate to https:/makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section on the page and select it.
4-Navigate to 'JavaScript' control and select it.
5-Navigate to various controls lying on the pane opened.
6-Launch Color Contrast Analyzer tool using WAT tool bar.
7-Verify, the luminosity ratio is greater than 4.5:1 for link text or not.  

**Actual Result** 
Luminosity ratio is less than 4.5:1 for all links text on JavaScript pane.

**Expected Result**
 Luminosity ratio should be more than 4.5:1 for all links text on JavaScript pane.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

Suggested fix
1.1-Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text 
1.2- If design allows , Not specify background color, not specify text color, and not using technology features that change those defaults .In this technique the author avoids having to do any contrast measures by simply not specifying the text color and not specifying the background. As a result the colors of both are completely determined by the user agent. 

**Tips:-**
1.The easiest fix is to increase the brightness of light colors and decrease the brightness of darker colors. 
2.Exaggerate lightness differences between foreground and background colors, and avoid using colors of similar brightness adjacent to one another, even if they differ in saturation or hue.
3.During design Use the Color Contrast Analyzer to determine whether new colors have color contrast ratio of at least 4.5:1.

**Key References for Fix**
http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
)![mas1 4 3_javascript_ improper luminosity ratio for link texts on java script pane](https://user-images.githubusercontent.com/30459830/28613865-4ed50cea-7211-11e7-87af-8d08e3edcb1f.png)

**Comment by v-amik** [comment]
review done

**Comment by v-mesala** [comment]
Verified on URL: https:/makecode.microbit.org/acc and issue is fixed now. Hence closing the bug
